fill/actuators/ignitor.cc

```diff
@@ -0,0 +1,21 @@
+#include "ignitor.h"
+
+Ignitor::Ignitor(){
+    wiringPiSetupGpio();
+    pinMode(IG_0, OUTPUT);
+    pinMode(IG_1, OUTPUT);
+
+    // write low initially 
+    digitalWrite(IG_0, LOW);
+    digitalWrite(IG_1, LOW);
+}
+
+Ignitor::~Ignitor(){}
+
+bool Ignitor::Actuate(){
```

Comment:
Have you confirmed with electrical that there is no situation where we'd want to only ignite one igniter? (Current limit?) IIRC, that is correct, but good to double check

Reply:
Just checked with Nandita - yeah only ever 2 at a time